util/gpg/gpg.go

```diff
@@ -149,7 +149,7 @@ const MaxVerificationLinesToParse = 40
 
 // Helper function to append GNUPGHOME for a command execution environment
 func getGPGEnviron() []string {
-	return append(os.Environ(), fmt.Sprintf("GNUPGHOME=%s", common.GetGnuPGHomePath()), "LANG=C")
+	return append(os.Environ(), fmt.Sprintf("GNUPGHOME=%s", common.GetGnuPGHomePath()), "LANG=c")
```

Comment:
I think this should be reverted as well.

Reply:
Yes, indeed. Good catch again. Sorry for being a little inattentive here